pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -1167,8 +1166,22 @@ protected void internalSplitNamespaceBundle(AsyncResponse asyncResponse, String
         });
     }
 
+    private String getBundleRange(String bundleName) {
+        if (BundleType.LARGEST.toString().equals(bundleName)) {
+            return findLargestBundleWithTopics(namespaceName).getBundleRange();
+        } else if (BundleType.HOT.toString().equals(bundleName)) {
+            return findHotBundle(namespaceName).getBundleRange();
```

Comment:
Should we check `findHotBundle(namespaceName)` is null?

Reply:
namespace by default initializes min 1 bundle so, it can't be null.